### PR TITLE
[New Rule] FirstTimeSeen User Performing DCSync

### PR DESCRIPTION
**Pull request by Samirbous**
DCSync detection based on 4662 event properties is very noisy and all attributes looks similar to legit ones. The only differentiator is the `SubjectUserName` performing the action, using `New Term` rule type we can improve the detection TP rate and reduce noise.

```
[rule.new_terms]
field = "new_terms_fields"
value = ["winlog.event_data.SubjectUserName"]

[[rule.new_terms.history_window_start]]
field = "history_window_start"
value = "now-15d"
```

### CODE REVIEW

**Review by imays11 — APPROVED** [review]
nice use of new_terms rule type for tuning 👍🏾

File: rules/windows/credential_access_dcsync_newterm_subjectuser.toml [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+Windows Settings >
+Security Settings >
+Advanced Audit Policies Configuration >
+Audit Policies >
+DS Access >
+Audit Directory Service Changes (Success,Failure)
+```
+
+If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2, events will not define `event.ingested` and default fallback for EQL rules was not added until 8.2, so you will need to add a custom pipeline to populate `event.ingested` to @timestamp for this rule to work.
+"""
```
Comment (w0rk3r) [inline rules/windows/credential_access_dcsync_newterm_subjectuser.toml:97]:
```suggestion
```

Let's remove this one by now

File: rules/windows/credential_access_dcsync_newterm_subjectuser.toml [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule identifies when a User Account starts the Active Directory Replication Process for the first time. 
+Attackers can use the DCSync technique to get credential information of individual accounts or the entire domain, 
+thus compromising the entire domain.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-system.*"]
```
Comment (w0rk3r) [inline rules/windows/credential_access_dcsync_newterm_subjectuser.toml:17]: Maybe add the WEF index?

File: rules/windows/credential_access_dcsync_newterm_subjectuser.toml [hunk 3]
```diff
@@ -0,0 +1,146 @@
+[metadata]
+creation_date = "2022/12/19"
+integration = ["windows"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup, New Term rule type"
```
Comment (w0rk3r) [inline rules/windows/credential_access_dcsync_newterm_subjectuser.toml:5]: I think we need to adjust this one, new terms was added in 8.3?

File: rules/windows/credential_access_dcsync_newterm_subjectuser.toml [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+    "https://threathunterplaybook.com/library/windows/active_directory_replication.html?highlight=dcsync#directory-replication-services-auditing",
+    "https://github.com/SigmaHQ/sigma/blob/master/rules/windows/builtin/security/win_ad_replication_non_machine_account.yml",
+    "https://github.com/atc-project/atomic-threat-coverage/blob/master/Atomic_Threat_Coverage/Logging_Policies/LP_0027_windows_audit_directory_service_access.md",
+    "https://attack.stealthbits.com/privilege-escalation-using-mimikatz-dcsync",
+    "https://www.thehacker.recipes/ad/movement/credentials/dumping/dcsync",
+]
+risk_score = 73
+rule_id = "5c6f4c58-b381-452a-8976-f1b1c6aa0def"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Credential Access", "Active Directory", "has_guide"]
```
Comment (w0rk3r) [inline rules/windows/credential_access_dcsync_newterm_subjectuser.toml:109]:
```suggestion
tags = ["Elastic", "Host", "Windows", "Threat Detection", "Credential Access", "Active Directory"]
```

**Review by w0rk3r — APPROVED** [review]

File: rules/windows/credential_access_dcsync_newterm_subjectuser.toml [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+description = """
+This rule identifies when a User Account starts the Active Directory Replication Process for the first time. 
+Attackers can use the DCSync technique to get credential information of individual accounts or the entire domain, 
+thus compromising the entire domain.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-system.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "FirstTime Seen Account Performing DCSync"
```
Comment (w0rk3r) [inline rules/windows/credential_access_dcsync_newterm_subjectuser.toml:20]:
```suggestion
name = "FirstTime Seen Account Performing DCSync"
note = """
## Setup

The 'Audit Directory Service Changes' logging policy must be configured for (Success, Failure).
Steps to implement the logging policy with Advanced Audit Configuration:

```
Computer Configuration >
Policies >
Windows Settings >
Security Settings >
Advanced Audit Policies Configuration >
Audit Policies >
DS Access >
Audit Directory Service Changes (Success,Failure)
```

If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2, events will not define `event.ingested` and default fallback for EQL rules was not added until 8.2, so you will need to add a custom pipeline to populate `event.ingested` to @timestamp for this rule to work.
"""
"""
```

Comment (w0rk3r) [reply rules/windows/credential_access_dcsync_newterm_subjectuser.toml:20]: There is a conflict on this one in the ``` usage

File: rules/windows/credential_access_dcsync_newterm_subjectuser.toml (hunk 2 above)
Comment (Samirbous) [reply rules/windows/credential_access_dcsync_newterm_subjectuser.toml]: ++ added

File: rules/windows/credential_access_dcsync_newterm_subjectuser.toml [hunk 6]
```diff
@@ -0,0 +1,87 @@
+[metadata]
+creation_date = "2022/12/19"
+integration = ["windows"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup, New Term rule type"
```
Comment (w0rk3r) [inline rules/windows/credential_access_dcsync_newterm_subjectuser.toml:5]:
```suggestion
min_stack_comments = "The New Term rule type used in this rule was added in Elastic 8.4"
```

File: rules/windows/credential_access_dcsync_newterm_subjectuser.toml [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+min_stack_version = "8.6.0"
```
Comment (w0rk3r) [inline rules/windows/credential_access_dcsync_newterm_subjectuser.toml:6]:
```suggestion
min_stack_version = "8.4.0"
```

**Review by w0rk3r — APPROVED** [review]

**Pull request merged**